service/etc/d-installer.yaml

```diff
@@ -12,3 +12,72 @@ web:
   ssl: null
   ssl_cert: null
   ssl_key: null
+
+storage:
+  volumes:
+  - mount_point: "/"
+    fs_type: btrfs
+    desired_size: 10 GiB
+    min_size: 5 GiB
+    max_size: unlimited
+    weight: 30
+
+    # There must always be a root
+    proposed_configurable: false
+
+    snapshots: true
+    snapshots_percentage: 250
+    snapshots_configurable: true
+    # Disable snapshots if there is not enough room
+    disable_order: 3
+
+    btrfs_default_subvolume: "@"
+    subvolumes:
+    - path: home
+    - path: opt
+    - path: root
+    - path: srv
+    - path: usr/local
+    # Unified var subvolume - https://lists.opensuse.org/opensuse-packaging/2017-11/msg00017.html
+    - path: var
+      copy_on_write: false
+
+    # Architecture specific subvolumes
+    - path: boot/grub2/arm64-efi
+      archs: aarch64
+    - path: boot/grub2/arm-efi
+      archs: arm
+    - path: boot/grub2/i386-pc
+      archs: x86_64
+    - path: boot/grub2/powerpc-ieee1275
+      archs: ppc,!board_powernv
+    - path: boot/grub2/s390x-emu
+      archs: s390
+    - path: boot/grub2/x86_64-efi
+      archs: x86_64
+    - path: boot/grub2/riscv64-efi
+      archs: riscv64
+
+  - mount_point: "/home"
+    fs_type: xfs
+    desired_size: 40 GiB
+    min_size: 10 GiB
+    max_size: unlimited
+    weight: 60
+
+    proposed: false
```

Comment:
I think we should at least link current existing documentation, as some keys looks strange for me.

Reply:
Of course, the plan was to link to the corresponding YaST documentation. Now it's done.